mergify_engine/engine/commands_runner.py

```diff
@@ -37,7 +37,7 @@
 
 LOG = daiquiri.getLogger(__name__)
 
-COMMAND_MATCHER = re.compile(r"^@Mergify(?:|io) (\w*)(.*)", re.IGNORECASE)
+COMMAND_MATCHER = re.compile(r"^(?:https://github.com/|@)Mergify(?:|io) (\w*)(.*)", re.IGNORECASE)
```

Comment:
Hum I think `:?` makes the URL prefix or the `@` optional. I'd rather keep them mandatory.

Reply:
No the `:?` is just to indicate that its a non-capturing group